jetty-websocket/websocket-core/pom.xml

```diff
@@ -134,9 +134,8 @@
                   <goal>fuzzingclient</goal>
                 </goals>
                 <configuration>
-                  <!-- The class which contains a main method that accept the port as parameter and startup the -->
+                  <!-- The class contains a main method that accept the port as parameter and startup the -->
```

Comment:
The class that contains a main method which accepts the port as a parameter to start the server